packages/react-components/react-tabs/README.md

```diff
@@ -2,4 +2,41 @@
 
 **React Tabs components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
 
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+- A `TabList` provides single selection from a list of tabs.
+- When a `Tab` is selected, the content associated with the selected tab is displayed and other content is hidden.
+- A `TabList` has options to control how tabs are dispayed:
+  - horizontal or vertical
+  - transparent or subtle appearance
+  - small or medium size tabs
+- Each `Tab` typically contains a text header and often includes an icon.
+
+## Usage
+
+To display tabs, declare a `TabList` with a list of `Tab` components as children.
+
+Text is typically used within each tab, but you can place any content you like.
+You can add an icon to a tab through the `icon` property.
+
+Each `Tab` requires a unique `value`.
+The value is passed as part of the data parameter when a tab is clicked and the `onTabSelect` event is raised.
+
+```tsx
+import { TabList, Tab } from '@fluentui/react-components';
+
+const onTabSelect = (event: SelectTabEvent, data: SelectTabData) => {
+  //show the content associated with the selected tab
```

Comment:
in the example it would help to show the shape of data, and how a tab value is returned. i know it's in the typings, but it'd help explain line 21 above